scio-test/src/test/scala/com/spotify/scio/coders/CoderTest.scala

```diff
@@ -391,19 +393,18 @@ final class CoderTest extends AnyFlatSpec with Matchers {
   it should "have a nice verifyDeterministic exception for case classes" in {
     val caught =
       intercept[NonDeterministicException] {
-        val coder = Coder[(Double, Double, Double)]
+        val coder = Coder[NonDeterministic]
 
         materialize(coder).verifyDeterministic()
       }
 
     val expectedMsg =
-      "RecordCoder[scala.Tuple3](_1 -> DoubleCoder, _2 -> DoubleCoder, _3 -> DoubleCoder)" +
+      "RecordCoder[com.spotify.scio.coders.NonDeterministic](a -> DoubleCoder, b -> DoubleCoder)" +
```

Comment:
This seems like a worse error message. Why change it ?

Reply:
This is actually test `case class` with non deterministic fields. `tuple` is being tested above.